ledger/sandbox/src/main/scala/com/digitalasset/platform/indexer/JdbcIndexer.scala

```diff
@@ -128,14 +131,51 @@ class JdbcIndexer private[indexer] (
   private var lastReceivedRecordTime: Long = Instant.now().toEpochMilli
 
   override def subscription(readService: ReadService): ResourceOwner[IndexFeedHandle] =
-    new SubscriptionResourceOwner(readService)
+    new SubscriptionResourceOwner(readService, batchSize)
 
-  private def handleStateUpdate(offset: Offset, update: Update): Future[Unit] = {
-    lastReceivedRecordTime = update.recordTime.toInstant.toEpochMilli
+  /**
+    * Turns a list of generic updates into a list where consecutive [[Update.TransactionAccepted]] updates
+    * are grouped in sub-lists.
+    */
+  private def splitUpdates(list: List[(Offset, Update)])
```

Comment:
Is this the right name? The comment says that this groups together consecutive accepted transactions.
```suggestion
  private def groupConsecutiveAcceptedTransactions(list: List[(Offset, Update)])
```